pkg/controller/cloneset/scale/cloneset_scale.go

```diff
@@ -55,9 +55,13 @@ func (r *realControl) ManageReplicas(
 	pods []*v1.Pod, pvcs []*v1.PersistentVolumeClaim,
 ) (bool, error) {
 	if updateCS.Spec.Replicas == nil {
-		return false, fmt.Errorf("spec.Replicas is nil")
+		// TODO: add webook mutate to set deault value to Replicas
+		updateCS.Spec.Replicas = utilpointer.Int32Ptr(0)
```

Comment:
In Statefulset,  the default is 1. // If unspecified, defaults to 1.

Should we use 1 as well?